Automattic/wp-calypso
# Hosting: Allow all sites to see the Hosting section in development.

Allow all sites to see the Hosting Config section in their menu; this requires enabling a feature flag by query param or environment variable. This will prepare us for presenting an upgrade path to those noticing the feature and wanting it for their currently ineligible site.

#### Testing instructions

* Start this PR in your local dev environment with `ENABLE_FEATURES=hosting/all-sites npm start`.
* Select a VIP site and a Jetpack site; verify neither has Manage > Hosting Configuration in the sidebar, nor access to the page directly by URL (`https://calypso.localhost:3000/hosting-config/:domain`).
* Select a Simple site that has no plan or the Premium plan.
* Verify Manage > Hosting Configuration appears in the sidebar, and that the page is accessible.
* Select a site Simple site on the Business plan; verify Hosting features are accessible.
* Select an Atomic site; verify Hosting features are accessible.

Fixes #36489
